observability-charter.md

```diff
@@ -0,0 +1,207 @@
+  trace spans, and profiling of cloud native workloads.
+- Using distributed trace tooling to observe a series of calls between
+  microservices to understand where time is being spent.
+- Managing the complexity, operational cost, and resource consumption of
+  observability tools and systems at the enterprise scale.
+- Best practices for meaningful alerting, queries, and operational dashboards
+  including how to manage things including rules, definitions, thresholds and
+  policies.
+- How developers, operators, SRE, IT, and other actors comprehend, process, and
+  reason on distributed cloud-native systems.
+- Projects that incorporate novel & insightful approaches to utilizing
+  observability data such as:
+  - ML, model training, Bayesian networks, and other data science techniques
+    that enable anomaly & intrusion detection.
+  - correlating resource consumption with costing data to reduce the total cost
+    of cloud native infrastructure
+  - Using observability data exposed by service meshes, orchestrators, and other
+    metric sources to inform continuous deployment tooling (e.g. Canary
+    Predicates/Judges).
+- Objective curation and generation of case studies pertaining to delivering
+  observability tools/systems to end users.
+- Best practices around observability and its continuous improvement, e.g. post
+  mortems, runbooks
+- Provide guidance around and foster interoperability between observability
+  solutions without trying to enforce one specific standard
+- Foster understanding of the prerequisites and corner-stones of observability
+  like SLI/KPI, service objectives, and internal/external commitments.
+
+The following is a non-exhaustive sample list of activities and deliverables
+that are in-scope for this SIG
+
+- Summary and overview of projects available in the community.
+- Catalog of reference architectures that draw from CNCF projects, combining
+  them in useful and novel ways.
+- Definitions of implementations and patterns for best practices for
+  delivering observability tooling at enterprise scale.
+- Tooling composition and tool chain creation based on existing projects.
+- Best practices for operations and monitoring workflows using CNCF Projects.
+- Organizing and helping to provide visibility to Meetups, Blogs, and Podcasts
+  related to the scope of the SIG.
+- Guidance for application development and architecture that is observable.
+- Replicatable reference architectures.
+- Patterns for observing application delivery pipelines.
+- Education regarding instrumentation cloud native workloads.
+- Processing and Accessing relevant observability data at scale.
+- Policy and security controls for observabilty data.
+- Creating artifacts as part of CI/CD pipelines that facilitate observation of
+  services.  Concrete examples might be:
+  - service profiles for Linkerd
+  - debug binaries or other diagnostic metadata.
+  - representative trace spans from failing CI tests.
```

Comment:
```suggestion
  - Representative trace spans from failing CI tests.
```

Reply:
Also not sure about this one... It sounds like it related to debugging CI/CD not to debug service that is released or deployed :thinking: